samples/net/https_client/README.rst

```diff
@@ -0,0 +1,124 @@
+.. _https-client-sample:
+
+HTTPS Client
+############
+
+Overview
+********
+
+This sample application shows how to create encrypted HTTP 1.1 requests
+to an HTTPS server and how to parse the incoming responses.
+Supported HTTP 1.1 methods are: GET, HEAD, OPTIONS and POST.
+
+The source code for this sample application can be found at:
+:file:`samples/net/https_client`.
+
+Requirements
+************
+
+- :ref:`Networking with Qemu <networking_with_qemu>`
```

Comment:
PR 415 is making the same change?   Same comment -- Qemu should be spelled QEMU.  If you just use :ref:`networking_with_qemu` the output will be the title of that section (which is "Networking with QEMU")